static/styles/prism.less

```diff
@@ -8,6 +8,7 @@
       background: #eeeeee;
       top: 0px;
       right: 0;
+      opacity: 1;

```

Comment:
What does these changes do? Any chance you can show a before & after?

Reply:
it will make the copy button of code blocks always visible (or any other button in the code block's top right)
before: button would show up on hover

after: always visible
![image](https://github.com/bitovi/academy/assets/26363255/7a10cacd-29eb-406c-beee-3a3e169e25bb)

Reply:
Ah, ok, I don’t think we want to do this as-is because it’ll cover text we want to show: 

<img width="684" alt="Screenshot of Copy button blocking the code underneath." src="https://github.com/bitovi/academy/assets/10070176/4228bb3e-97ed-4060-9c27-50342846ef96">

Maybe we could move it to be on top of the code block? We could use some design help here, but here’s an idea:

<img width="653" alt="Screenshot of Copy button aligned on top of the code block so it doesn’t block the text underneath." src="https://github.com/bitovi/academy/assets/10070176/fd31c40e-6fc4-49dd-bc72-522da72c1e0f">

Reply:
updated, here's an example:
![image](https://github.com/bitovi/academy/assets/26363255/e6a014b1-484e-4f38-81da-cfe45d9e1cb2)